src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta": {
+    "releasedOn": "2023.09.13"
+  },
+  "added": [
+    {
+      "what": [
+        "splice function in workflows that can change the contents of a collection of elements by removing or replacing existing elements and/or adding new elements in place."
```

Comment:
We should use the same terminology as we use in workflows; we use arrays, not collections. If you're referring to strings also, clarify.